modules/react/card/lib/Card.tsx

```diff
@@ -33,7 +45,31 @@ export const Card = createComponent('div')({
     );
   },
   subComponents: {
+    /**
+     * `Card.Heading` is an optional subcomponent that is meant to describe the Card. Since `Card`
+     * is a non-semantic presentational component, `Card.Heading` does not automatically have any
+     * semantic meaning. If your use case requires the Heading to label the Card, you must do so
+     * manually.
+     *
+     * For example, {@link Modal} (which uses a `Card`) adds an `aria-labelledby` and a `role` to
+     * the `Card`, as well as an `id` to the `Card.Heading`.
+     *
+     * ```tsx
+     * <Card role="dialog" aria-labelledby="card-heading">
+     *   <Card.Heading id="card-heading">Card Title</Card.Heading>
+     *   <Card.Body>Card Contents</Card.Body>
+     * </Card>
+     * ```
+     *
+     * `Card.Heading` defaults to an `<h3>` element, but it can be changed using the `as` prop.
+     */
     Heading: CardHeading,
+    /**
+     * `Card.Body` is a non-semantic subcomponent that contains the body of the card. Attributes may
+     * be added to give `Card.Body` extra semantic meaning. If `Card.Body` is brief (like in a short
```

Comment:
"extra" implies it has semantic meaning already but it sounds like it doesn't.

```suggestion
     * be added to give `Card.Body` semantic meaning. If `Card.Body` is brief (like in a short
```
Check Prettier formatting for length if applied.